app/assets/javascripts/maps.js

```diff
@@ -0,0 +1,31 @@
+function displayMap(address) {
+    var addressObj = $.get( 'https://maps.googleapis.com/maps/api/geocode/json?address='+address);
+
+    var the_url = 'https://maps.googleapis.com/maps/api/geocode/json?address='+address
```

Comment:
Line is too long.<br>Identifier 'the_url' is not in camel case.<br>Missing semicolon.